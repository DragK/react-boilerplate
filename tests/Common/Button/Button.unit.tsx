import * as React from 'react'
import { mount } from 'enzyme'
import { Button } from '../../../src/Common/index'

describe('Button' , () => {
    it('it renders button ' , () => {
        const wrapper = mount(<Button label='test'/>)
          expect(wrapper).toHaveLength(1)
    })

    it('it match with snapshot' , () => {
        const wrapper = mount(<Button label='test'/>)
        expect(wrapper).toMatchSnapshot()
    })
})

