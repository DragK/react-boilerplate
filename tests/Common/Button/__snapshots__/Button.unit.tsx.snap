// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button it match with snapshot 1`] = `
<Button
  label="test"
>
  <button>
    test
  </button>
</Button>
`;
